CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ v1.8.0 [unreleased]
 - [#283](https://github.com/latera/homs/pull/283) Seed improvements: add env variables for initial values
 - [#288](https://github.com/latera/homs/pull/288) Add drag and drop field for file upload.
 - [#309](https://github.com/latera/homs/pull/309) Add error boundaries to all form components.
+- [#315](https://github.com/latera/homs/pull/315) Add additional exceptions for Oracle's date formats.
```

Comment:
We are implementing this fix especially for stable version. For future versions we have other great plan.